addons/project/report/project_report.py

```diff
@@ -34,8 +34,7 @@ class ReportProjectTaskUser(models.Model):
     rating_avg = fields.Float('Average Rating', readonly=True)
     priority = fields.Selection([
         ('0', 'Low'),
-        ('1', 'Normal'),
-        ('2', 'High')
+        ('1', 'High')
```

Comment:
Looks strange that we kept the previous values in the reporting. I thought it was for compatibility issues but it does not seem to be the case since I did made some research and I found several upgrade scripts on this field with the last one in version 8 where it should handle the migration from the previous values to the new one. So we should be safe 🟢